debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/LegacyV1CassandraSourceInfoStructMaker.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.cassandra;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.connector.LegacyV1AbstractSourceInfoStructMaker;
+import io.debezium.connector.cassandra.transforms.CassandraTypeKafkaSchemaBuilders;
+import io.debezium.util.SchemaNameAdjuster;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+
+public class LegacyV1CassandraSourceInfoStructMaker extends LegacyV1AbstractSourceInfoStructMaker<SourceInfo> {
```

Comment:
See my comment above; shouldn't be needed.

Reply:
> See my comment above; shouldn't be needed.

Removed LegacyV1CassandraSourceInfoStructMaker accordingly.